examples/darwin-framework-tool/commands/clusters/ReportCommandBridge.h

```diff
@@ -199,34 +214,59 @@ class SubscribeEvent : public ModelCommand {
         dispatch_queue_t callbackQueue = dispatch_queue_create("com.chip.command", DISPATCH_QUEUE_SERIAL);
 
         MTRSubscribeParams * params = [[MTRSubscribeParams alloc] initWithMinInterval:@(mMinInterval) maxInterval:@(mMaxInterval)];
+        if (mEventNumber.HasValue()) {
+            params.eventMin = [NSNumber numberWithUnsignedLongLong:mEventNumber.Value()];
+        }
         if (mKeepSubscriptions.HasValue()) {
             params.replaceExistingSubscriptions = !mKeepSubscriptions.Value();
         }
+        if (mIsUrgent.HasValue()) {
+            params.isUrgentEvent = mIsUrgent.Value();
+        }
         if (mAutoResubscribe.HasValue()) {
             params.resubscribeIfLost = mAutoResubscribe.Value();
         }
 
-        [device subscribeWithQueue:callbackQueue
-            params:params
-            clusterStateCacheContainer:nil
-            attributeReportHandler:^(NSArray * value) {
-                SetCommandExitStatus(CHIP_NO_ERROR);
-            }
-            eventReportHandler:^(NSArray * value) {
-                for (id item in value) {
-                    NSLog(@"Response Item: %@", [item description]);
+        if (mClusterId != chip::kInvalidClusterId) {
+            [device subscribeToEventsWithEndpointID:[NSNumber numberWithUnsignedShort:endpointId]
+                clusterID:[NSNumber numberWithUnsignedInteger:mClusterId]
+                eventID:[NSNumber numberWithUnsignedInteger:mEventId]
```

Comment:
If this is meant to be API-compatible with chip-tool, this should probably treat the invalid id values as wildcards...  Followup for doing that, please.

Reply:
@bzbarsky-apple This is current status. If more changes are needed, please let us know:
1. Arguments of 0xFFFF is working as wildcards like chip-tool
2. To support more wildcard logics same to chip-tool, "any" cluster is added

Reply:
> Arguments of 0xFFFF is working as wildcards like chip-tool

Sort of by accident, because the Darwin framework does not reject them as invalid values.  Which it probably should, because those are not valid ids.  The way wildcards are represented in the Darwin API is with `nil`, not magic invalid values....